frontend/src/app/modules/common/form-field/form-field.component.ts

```diff
@@ -16,32 +18,38 @@ import {
 export class OpFormFieldComponent {
   @HostBinding('class.op-form-field') className = true;
   @HostBinding('class.op-form-field_invalid') get errorClassName() {
-    return this.isInvalid;
+    return this.showErrorMessage;
   }
 
   @Input() label = '';
+  @Input() inlineLabel = true;
```

Comment:
There's a couple of issues with this imo;

1. I think a name like `wrapLabel` or even `wrapLabelAroundInput` is be more clear in what it does. `inline` could mean `display: inline` , but with no change in functionality.
2. If I understand correctly, this defaults to a different behaviour than the current component, which would constitute a breaking change. I think we should default to the current behavior (label wraps input) because the cases where you would not want that to happen are the minority (basically only the ckeditor, afaik)

Reply:
Line 25 is wrong, it should be false/undefined, thanks for spotting that. I agree that ``wrapLabel`` would be much more clear but I wanted to avoid having to always pass the @Input for the default behavior so I created the @Input for the exception (the formattable field). In any case, 'inlineLabel' is not 100% clear, any suggestion on that?

Reply:
`noWrapLabel` might be fine, it's in the style of `novalidate` or `noreferrer`, which are HTML staples